hip-0000-nft-music-metadata.md

```diff
@@ -0,0 +1,252 @@
+|                                         | key                     |          | Key                     | String                | The musical key of the audio track.                                                                                |
+|                                         | location_created        |          | Location Created        | String                | The location where the audio track was created                                                                     |
+|                                         | recording_studio        |          | Record Studio           | String                | The recording studio where the track was recorded                                                                  |
+|                                         | license                 |          | License                 | String                | The type of license associated with the audio track.                                                               |
+|                                         | copyright               |          | Copyright               | String                | The type of license copyright with the audio track.                                                                |
+|                                         | label                   |          | Label                   | String                | The name of the record label that released the audio track.                                                        |
+|                                         | master                  |          | Master (Rights)         | Array                 | The name of the company holding the master recording rights                                                        |
+|                                         | publisher               |          | Publisher (Rights)      | Array                 | The name of the publishing company that holds publishing and performance rights                                    |
+|                                         | pro                     |          | PRO (Rights)            | Array                 | Performance Rights Organization(s) or society(s) associated with mechanical rights                                 |
+|                                         | release_date            |          | Release Date            | String                | The release date of the audio track.String (date format: "UTC timestamp")                                          |
+|                                         | isrc                    |          | ISRC                    | String                | The International Standard Recording Code (ISRC) associated with the audio track.                                  |
+|                                         | lyrics                  |          | Lyrics                  | String                | The time-stamped lyrics for the audio track (symbols in UNICODE), ex: [00:00:00] lyric here [00:00:05] lyric there |
```

Comment:
Instead of having lyrics inline
```json
"lyrics": "[00:00:00] Echoes in the circuitry... [00:00:30] A digital awakening...",
```
it might be better to simply reference a file instead, using `String (URI)` and perhaps also `String (MIME type)`

The contents of this file could conform to any one of a set of a whitelisted set of file formats that are already in use (and therefore this HIP would not need to define, and for which existing software libraries could be used to parse). e.g. [lrc](https://en.wikipedia.org/wiki/LRC_(file_format)), or even [srt](https://en.wikipedia.org/wiki/SubRip#Format).

Reply:
i actually really like this. it seems like a pretty simple change and cleans up the metadata a lot. what do you think of this @ty-swirldslabs

Reply:
hey @bguiz just following up here. the group has determined that inline lyrics would be the ideal approach to minimize requests and simplify the composition of the metadata (in the music world, metadata complexity is expected, for some thousands of lines).

an option to explore would be "if MIME look for URI" but that would add complexity. for the scope of this, i think inline lyrics suit the purpose.